Add Unforgiving Devices Patches

IHateMyKite opened the pull request.

IHateMyKite committed:
Add Unforgiving Devices Patches

sibir-ine commented on masterlist.yaml:
```diff
@@ -28190,3 +28190,49 @@ plugins:
         crc: 0x9E8F3CA5
         util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 17
+        
```
There are some extra spaces here:

```suggestion

```

sibir-ine commented on masterlist.yaml:
```diff
@@ -28190,3 +28190,49 @@ plugins:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'unforgiving Devices DD Patch'
+    after:
+      - 'Devious Devices SE patch.esp'
+      - 'Devious Devices - Assets.esm'
+      - 'Devious Devices - Expansion.esm'
+      - 'Devious Devices - Integration.esm'
+      - 'Devious Devices For Him.esp'
+      - 'Devious Devices - BRRF.esp'
+      - 'Devious Devices - Lore Patch.esp'
```
There are a few things to note here about LOOT's sorting. If Plugin A has Plugin B as a master, LOOT will automatically sort Plugin A after Plugin B with no load after rule required. Also, if Plugin C is a master plugin while Plugin D isn't, Plugin D will automatically be loaded after Plugin C as that rule is also hard-coded by the game. If either of these are the case for any of these plugins, please remove them from the list as they are redundant. Otherwise, they can be left, but please sort the remaining plugins alphabetically (ignoring symbols & spaces). All of this applies to the other load after rules in this PR as well, so please double-check them.

sibir-ine commented on masterlist.yaml:
```diff
@@ -28190,3 +28190,49 @@ plugins:
+        name: 'unforgiving Devices Lauras Bondage Shop Patch'
+    after:
+      - 'UnforgivingDevices - DD Patch.esp'
+      - 'Laura's Bondage Shop.esp'
```
Apostrophes need to be escaped (or this plugin should be removed if it's redundant):

```suggestion
      - 'Laura''s Bondage Shop.esp'
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -28190,3 +28190,49 @@ plugins:
+      - 'Deviously Cursed Loot.esp'
+    clean:
+      - crc: 0xB694000E
+        util: 'SSEEdit v4.0.4'
```
There should be an extra newline at the end of the file:

```suggestion
        util: 'SSEEdit v4.0.4'

```

IHateMyKite committed:
Update

- Removed extra spaces.
- Removed redundant code
- Added new line at the end of file
- Added missing patches

sibir-ine commented on masterlist.yaml:
```diff
@@ -28190,3 +28190,59 @@ plugins:
+        name: 'Unforgiving Devices DD Patch'
+    after:
+      - 'Devious Devices SE patch.esp'
+      - 'Devious Devices - Lore Patch.esp'
```
These should be the other way around:

```suggestion
      - 'Devious Devices - Lore Patch.esp'
      - 'Devious Devices SE patch.esp'
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -28190,3 +28190,59 @@ plugins:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'Unforgiving Devices Lauras Bondage Shop Patch'
+    after:
+      - 'UnforgivingDevices - DD Patch.esp'
```
Single-element lists should be written in flow style:

```suggestion
    after: [ 'UnforgivingDevices - DD Patch.esp' ]
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -28190,3 +28190,59 @@ plugins:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'Unforgiving Devices Devious Pink Patch'
+    after:
+      - 'UnforgivingDevices - DD Patch.esp'
```
```suggestion
    after: [ 'UnforgivingDevices - DD Patch.esp' ]
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -28190,3 +28190,59 @@ plugins:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'Unforgiving Devices Security Overhaul Patch'
+    after:
+      - 'UnforgivingDevices - DD Patch.esp'
```
```suggestion
    after: [ 'UnforgivingDevices - DD Patch.esp' ]
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -28190,3 +28190,59 @@ plugins:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'Unforgiving Devices Devious Cursed Loot Patch'
+    after:
+      - 'UnforgivingDevices - DD Patch.esp'
```
```suggestion
    after: [ 'UnforgivingDevices - DD Patch.esp' ]
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -28190,3 +28190,59 @@ plugins:
+    url:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'Unforgiving Devices CACO Patch'
+    after:
+      - 'UnforgivingDevices - DD Patch.esp'
```
```suggestion
    after: [ 'UnforgivingDevices - DD Patch.esp' ]
```

IHateMyKite committed:
Reordered after list. Remade single element after to flow style

sibir-ine commented:
Before I merge this, what version number of the mod (or individual plugins if applicable) is the cleaning data for?

IHateMyKite commented:
All patches are for version of Unforgiving Devices 2.1. They patch following versions

- UnforgivingDevices - DD Patch.esp - 5.1 or 5.2 (there are 2 same named patches for different version selected with fomod). The cleaning was generated for 5.2 version.
- UnforgivingDevices - LBS Patch.esp - V3.42
- UnforgivingDevices - Pink Patch.esp - 0.3 beta
- UnforgivingDevices - Security Overhaul Patch.esp - 3.2.0
- UnforgivingDevices - DCL Patch.esp - 9.0
- UnforgivingDevices - CACO Patch.esp - 2.1.1

sibir-ine commented:
I meant the version numbers of the patches added by the PR.

IHateMyKite commented:
They are not versioned, as they are always part of the main mod fomod. So they can have same version as main mod -> 2.1
